src/ConnectedMode/Migration/Wizard/MigrationWizardWindow.xaml

```diff
@@ -77,16 +79,14 @@
         </DockPanel>
 
         <DockPanel x:Name="MigrationProgressWindow" Margin="10" LastChildFill="True" Visibility="Hidden">
-            <!-- Using a grid here because we want to draw one button on top of the other, and only have one visible at time -->
-            <Grid DockPanel.Dock="Bottom">
-                <Button HorizontalAlignment="Right" Content="Migrate" x:Name="migrateButton" Visibility="Visible" Click="OnStartMigration" Style="{DynamicResource Button_Style}"/>
-                <Button HorizontalAlignment="Right" Content="Finish" x:Name="finishButton" Visibility="Collapsed" IsCancel="True" Style="{DynamicResource Button_Style}"/>
-            </Grid>
+            <DockPanel DockPanel.Dock="Bottom">
+                <Button HorizontalAlignment="Right" Content="Finish" x:Name="finishButton" IsEnabled="false" IsCancel="True" Style="{DynamicResource Button_Style}"/>
+            </DockPanel>
 
             <!-- Last child - fills the body of the panel -->
             <DockPanel LastChildFill="True">
                 <TextBlock DockPanel.Dock="Top">Progress:</TextBlock>
-                <ListBox x:Name="progressList" Margin="0,8,0,10" BorderBrush="LightGray" ScrollViewer.VerticalScrollBarVisibility="Auto" />
+                <ListBox x:Name="progressList" Margin="0,8,0,10" BorderBrush="LightGray" ScrollViewer.VerticalScrollBarVisibility="Auto" Height="350" />
```

Comment:
Fixed height to stop it resizing the dialogue. A vertical scrollbar will appear if needed.